woocommerce/woocommerce-android
# Activate: Use the plan data to make make a purchase

I forgot to make these changes to the plan purchase after fetching plan data. Previously, we used hardcoded `product_id` values in FluxC to specify the eCommerce plan purchase. Now, we use the fetched plan data.

**Note:** There is an associated [FluxC PR](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2585).

<image src="https://user-images.githubusercontent.com/1522856/205100030-b79d8c65-e790-4cbc-91aa-39a32bdef566.png" width="300" />

**To test:**
It's enough to just start the store creation flow and verify that the monthly eCommerce plan is added to the webview checkout page.
